src/redist/targets/GenerateArchives.targets

```diff
@@ -20,25 +20,12 @@
         DestinationArchive="$(ArtifactsShippingPackagesDir)$(ArtifactNameWithVersionCombinedHostHostFxrFrameworkSdk).zip"
         OverwriteDestination="true" />
 
-    <ZipFileCreateFromDirectory
-        Condition=" '$(OSName)' == 'win' "
-        SourceDirectory="$(SdkInternalLayoutPath)"
-        DestinationArchive="$(ArtifactsNonShippingPackagesDir)$(ArtifactNameWithVersionSdk).zip"
-        OverwriteDestination="true" />
```

Comment:
Isn't that needed anymore in the current build?

Reply:
We need the layout, but not the archives.

Reply:
We don't need the archives anymore at all? Like in any kind of build?